subjects/java/checkpoints/todo-list/README.md

```diff
@@ -0,0 +1,68 @@
+## TodoList
+
+### Instructions
+
+Write a class `Task` that has two attributes: `description` and `status`, with the corresponding getters and setters.
+
+Write a class called `TodoList` that represents a to-do list. The class should be able to hold a specified number of tasks, each with a description and a status. Include methods to add tasks, set a task's status, update task descriptions, and display all tasks in a neat format showing their descriptions and statuses.
+
+> The description lenght will be less or equal to 30 Character.
+
+### Expected Functions
+
+```java
+enum TaskStatus {
+    NEW, IN_PROGRESS, COMPLETED
+}
+
+public class Task {
+    private String description;
+    private TaskStatus status;
+
+    public void setDescription(String description);
+    // Getters and Setters
+    // ...
+}
+
+public class TodoList {
+    private Task[] tasks;
+    private int capacity;
+
+    public TodoList(int capacity);
+
+    public void addTask(String description);
```

Comment:
it should stop adding new tasks

Reply:
It would be nice to specify the expected behaviour for these cases: in this case, nothing should happen. Since it's quite vague, I would be thinking: should I throw an error, print a message, or, more generally have to handle the error somehow?

Same for [this](https://github.com/01-edu/public/pull/2577#discussion_r1669505167)